add separate logic for vcenter conv-snap

naumvd95 opened the pull request.

naumvd95 committed:
add separate logic for vcenter conv-snap

rkhozinov commented on servers/partner-ci/common.yaml:
```diff
@@ -341,8 +341,11 @@
                 echo "snapshot_id_number=" $snapshots_id
-                python plugin_test/fuel-qa/utils/jenkins/conv_snapshot_file.py
-
+                if [ "${ENV_PREFIX}" == *"vcenter"* ]; then
```
[[ $SNAPSHOTS_ID == *"lastSuccessfulBuild"* ]]

naumvd95 committed:
using right-way to string comparisons

rkhozinov commented on servers/partner-ci/common.yaml:
```diff
@@ -341,8 +341,11 @@
                 echo "snapshot_id_number=" $snapshots_id
-                python plugin_test/fuel-qa/utils/jenkins/conv_snapshot_file.py
-
+                if [[ "${ENV_PREFIX}" == *"vcenter"* ]]; then
```
```
CONV_PATH="plugin_test/fuel-qa/utils/jenkins"
[[ "${ENV_PREFIX}" == *"vcenter"* ]] && CONV_PATH="utils/jenkins"
python "${CONV_PATH}/conv_snapshot_file.py"

```

rkhozinov commented:
or you can use special variable in common_props to define path to the conv.py

rkhozinov commented:
@naumvd95 see the comment ^

naumvd95 committed:
wrapping define-case

results=http://jenkins-tpi.bud.mirantis.net:8080/view/9.x.vcenter/job/9.x.vcenter.custom.qa/23/console

rkhozinov commented on servers/partner-ci/common.yaml:
```diff
@@ -341,8 +342,7 @@
                 echo "snapshot_id_number=" $snapshots_id
-                python plugin_test/fuel-qa/utils/jenkins/conv_snapshot_file.py
-
+                python "${CONV_SNAP_PATH}/conv_snapshot_file.py"
```
Let's define full path to the util:
`CONV_SNAP_PATH=utils/jenkins/conv_snapshot_file.py`

naumvd95 replied:
done

Commit:
vcenter,common: full path to the conv.py util
